cmd/kyverno/main.go

```diff
@@ -350,6 +350,7 @@ func main() {
 		log.Log.WithName("PolicyController"),
 		policyControllerResyncPeriod,
 		metricsConfig,
+		configuration,
```

Comment:
IIRC, the configuration is available globally.

Reply:
I'll check this again. Thanks